src/transformers/models/resnet/configuration_resnet.py

```diff
@@ -65,7 +65,7 @@ class ResNetConfig(PretrainedConfig):
 
     >>> # Initializing a ResNet resnet-50 style configuration
     >>> configuration = ResNetConfig()
-    >>> # Initializing a model from the resnet-50 style configuration
+    >>> # Initializing a model (with random weights) from the resnet-50 style configuration
```

Comment:
Can we have some new empty lines, just the ones you can find in other config files 🙏 , thanks

Reply:
@ydshieh made the suggested changes; good to go now :)